stubs/docutils/docutils/nodes.pyi

```diff
@@ -178,11 +179,66 @@ class Labeled: ...
 
 # Root Element
 
+_Document: TypeAlias = document
+_Decoration: TypeAlias = decoration
+
 class document(Root, Structural, Element):
-    parent: None
-    transformer: Transformer
-    def __init__(self, settings, reporter: Reporter, rawsource: str = "", *children: Node, **attributes) -> None: ...
-    def __getattr__(self, __name: str) -> Incomplete: ...
+    current_source: str = ...
```

Comment:
Not a full review, but please drop the `= ...` from class attributes. Our tests didn't catch that due to PyCQA/flake8-pyi#98.

Reply:
i've removed the `= ...` from class attributes. Looks like each of the method arguments also has a similar pattern. Are these required? (i've largely lifted this from another source so i'm just cargo-culting these changes)

Reply:
`= ...` is required for method arguments that have a default that we can't represent in the stubs. If it is representable (e.g. if the default is a simple value like `None`), we use that instead.

Reply:
ah perfect. have updated